docs/src/main/sphinx/security/ranger-access-control.md

```diff
@@ -0,0 +1,132 @@
+# Apache Ranger access control
+* Apache Ranger access control plugin supports use of Apache Ranger policies to authorize data access in Trino - like operations on catalogs, schemas, tables, columns.
+* Column-masking and row-filtering are supported by the plugin.
+* Accesses authorized by the plugin are audited for compliance purposes.
+
+## Requirements
+* Access to a Apache Ranger deployment having authorization policies to be enforced
+* Access to audit stores (Solr/HDFS/Log4J/S3) to save access audit logs
+* Apache Ranger 2.5.0 includes Trino service-def required by this access control plugin. In earlier versions of Apache Ranger, Trino service-def will have to be updated to the version [here](
+  https://github.com/apache/ranger/blob/ranger-2.5/agents-common/src/main/resources/service-defs/ranger-servicedef-trino.json).
+
+## Configuration
+Setup following configuration files to configure Apache Ranger as the authorizer in Trino:
+
+### /etc/trino/access-control.properties
+```properties
```

Comment:
Before the snippet also detail where that file should deployed .. on all Trino cluster nodes.

Reply:
Not sure .. could you test to verify so we get the docs correct?

Reply:
@mosabua  -  ok. Existing documentation on [System access control](https://trino.io/docs/current/security/built-in-system-access-control.html) indeed clarifies that:`Access control must be configured on the coordinator.`

Let me know if this should be repeated in Apache Ranger access control documentation page as well.

Reply:
Sure .. cant hurt and will help some users.